### PR TITLE
Feature/ppl export

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: client/components/download-header.js [hunk 1]
```diff
@@ -1,14 +1,64 @@
...
+  const container = useRef(null);
+  const download = useRef(null);
+  basename = basename.replace(/\/edit/, '');
+
+  // project title could span multiple lines, adjust download position accordingly
+  useEffect(() => {
+    // subtract padding, and border
+    const height = container.current.offsetHeight - 30 - 4;
+    download.current.style.height = `${height}px`;
+    download.current.style.lineHeight = `${height}px`;
```
Comment (lennym) [inline client/components/download-header.js]: Is this setting inline styles? Does that actually work with CSP policies as they are?

Comment (lennym) [reply client/components/download-header.js]: Spoiler: it shouldn't.

Comment (joefitter) [reply client/components/download-header.js]: it doesn't, updating the dom element in this way is fine with CSP, it's only if you set inline styles on the element using the style attribute

Comment (joefitter) [reply client/components/download-header.js]: also this is client side, so CSP has already done it's shizzle by the time this runs

Comment (lennym) [reply client/components/download-header.js]: I'm not sure that last bit is true - CSP should prevent new elements which violate the policy being added, but if it works then I'm happy.

Comment (joefitter) [reply client/components/download-header.js]: ORLY? I did not know that. However this does work

**Pull request merged**